mobify/platform-scaffold
# WEB-1504: Fix scss loader in webpack/base.onboarding.js

This PR fixes the scss loader in base.onboarding.js to allow .png files to be referenced in scss. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1504
 **Linked PRs**: n/a

## Changes
- Update scss loader in base.onboarding.js

## Todos

## How to test-drive this PR
- add a css rule to the project that references a .png file in the build
```
.c-test {
    background-image: url('./static/img/app-icon.png');
}
```
- Run `npm start`
- the PWA should build without any errors
